### ISSUE TITLE
Use Postponed Evaluation of Annotations

### ISSUE THREAD

**Issue by felix-andreas** [andreasfelix/apace#20]
> :warning: Currently blocked by PyPy as it currently only supports Python 3.6.

Use [Postponed Evaluation of Annotations](https://www.python.org/dev/peps/pep-0563/) for Cell in the classes module. 

This will become default behavior in Python 4.0 and is available for Python 3.7+ by importing the future module:
```python
from __future__ import annotations
```
